lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -16,8 +16,15 @@ late CanvasKit canvasKit;
 /// static APIs.
 ///
 /// See, e.g. [SkPaint].
+///
+/// This also acts as a cache of an initialized CanvasKit instance. We can use
+/// this, for example, to perform a hot restart without needing to redownload
+/// and reinitialized CanvasKit.
+@JS('window.flutterCanvasKit')
+external set windowFlutterCanvasKit(CanvasKit? value);
+
 @JS('window.flutterCanvasKit')
-external set windowFlutterCanvasKit(CanvasKit value);
+external CanvasKit? get windowFlutterCanvasKit;
```

Comment:
Should we use a more generic name so this method is compatible with Cobalt?

Reply:
I purposely left it open because it's unknown if we will need to call `CanvasKitInit` on the Cobalt implementation. So far now this is flutter-specific because it should be an already-initialized CanvasKit instance. If it turns out that the native CanvasKit instance is already initialized, we can rename this to be the same as the native CanvasKit object.